test/restaurant_inspections_api/handlers/counties_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns restaurant-inspections-api.handlers.counties-test
+  (:require [restaurant-inspections-api.handlers.counties :refer :all]
+            [clojure.test :refer :all]))
+
+(def ctx-mock {:request {:params {}}})
+(def response-mock {:meta {}
+                    :data [{:district "D1"
+                            :countynumber 23
+                            :countyname "Dade"
+                            :inspections 20630}]})
+;(deftest handle-ok-test
```

Comment:
This is one of the things I need some more time.
I added the test file because in theory if we add a handler we should add a test, but in the counties case we don't do anything but directly call the query. The way of testing this is stubbing the query fn to spy on it and maybe return a result, but I need more time to figure out how to do that. In this case I just commented the test to get back to it later.